source/Operation Guides/InterFlux Masternodes/Using Masternode Setup Tool/masternode-setup-tool.rst

```diff
@@ -0,0 +1,309 @@
+************
+Introduction
+************
+
+This guide details the process required for joining the Cirrus
+Sidechain as a Block Producing Node, known as an InterFlux Masternode using the Masternode Setup Tool. The request to join the Cirrus Sidechain as a block
+producer, requires you to setup the Masternode Setup tool, therefore, a request to join the Cirrus Sidechain will be
+made once the Masternode is fully configured.
+
```

Comment:
Can we add a section detailing what the Masternode Setup Tool is and how it incorporates all needed pre-requisites, resulting in a single-click application that guides through registration and running a Masternode?